tools/housekeeping

```diff
@@ -12,9 +12,6 @@ set -o errexit -o nounset -o pipefail
 # Regex to select files for clang-format
 readonly clangFormattableFiles='\.(h|c|hpp|cpp)$'
 
-# Regex to exclude files from clang-format
-readonly excludeFromClangFormat='(include/cse/fmuproxy|src/cpp/fmuproxy)/(fmu_service|service_types)'
```

Comment:
You should remove the line where `excludeFromClangFormat` is being used further down too.